### PR TITLE
[Fix] Privilege Reconstruction Data Query Fix & Resource Creation Post Oper…

### LINKED ISSUE #10312: [Permission] Resource permission module query page error.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Resource permission module query page error.

### What you expected to happen

normal

### How to reproduce

query resource.

### Anything else

No response

### Version

3.0.0-beta-1

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by WangJPLeo**
…ations.

## Purpose of the pull request

Privilege Reconstruction Data Query Fix & Resource Creation Post Operations.

## Brief change log

close #10312 

## Verify this pull request
Manually verified the change by testing locally.

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/BaseServiceImpl.java [hunk 1]
```diff
@@ -47,6 +48,15 @@ public class BaseServiceImpl implements BaseService {
     @Autowired
     protected ResourcePermissionCheckService resourcePermissionCheckService;
 
+    @Override
+    public void permissionPostHandle(AuthorizationType authorizationType, Integer userId, List<Integer> ids, Logger logger) {
+        try{
+            resourcePermissionCheckService.postHandle(authorizationType, userId, ids, logger);
+        }catch (Exception e){
+            logger.error("post handle error", e);
```
Comment (caishunfeng) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/BaseServiceImpl.java:56]: Is it better to throw a permission exception here?

File: dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/permission/ResourcePermissionCheckServiceImpl.java [hunk 2]
```diff
@@ -88,6 +88,11 @@ public boolean functionDisabled() {
         return false;
     }
 
+    @Override
+    public void postHandle(AuthorizationType authorizationType, Integer userId, List<Integer> ids, Logger logger) {
+        logger.info("no post handle");
```
Comment (caishunfeng) [inline dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/permission/ResourcePermissionCheckServiceImpl.java]:
```suggestion
        logger.debug("no post handle");
```

File: dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/permission/ResourcePermissionCheckService.java [hunk 3]
```diff
@@ -58,4 +58,12 @@
      * @return
      */
     boolean functionDisabled();
+
+    /**
+     * postHandle
```
Comment (caishunfeng) [inline dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/permission/ResourcePermissionCheckService.java]: add some doc

**Review by lenboo — APPROVED** [review]
+1

**Pull request merged**